compiler/crates/js-config-loader/src/loader.rs

```diff
@@ -62,6 +62,10 @@ impl<T: for<'de> Deserialize<'de> + 'static> Loader<T> for JsonLoader {
 pub struct JsLoader;
 impl<T: for<'de> Deserialize<'de> + 'static> Loader<T> for JsLoader {
     fn load(&self, path: &Path) -> Result<Option<T>, ErrorCode> {
+        // Windows requires absolute paths to be valid file:// URLs
```

Comment:
Are we guaranteed to get absolute paths here? Can node deal with relative `file://` URLs? If not we need to detect which case we're in.